aries-site/src/pages/templates/global-banner-notifications.mdx

```diff
@@ -30,7 +31,7 @@ import {
 
 Global banners are high-attention notifications used primarily for status updates on background processes and upcoming system changes and releases.
 
-Global banners are used to showcase warning and critical statuses or informational alerts. Depending on the severity of the information, they may remain persistent across multiple sessions or be dismissible by the user.
+Global notifications are used to showcase warning and critical statuses or informational alerts. Depending on the severity of the information, they may remain persistent across multiple sessions or be dismissible by the user.
```

Comment:
```suggestion
Global banner notifications are used to showcase warning and critical statuses or informational alerts. Depending on the severity of the information, they may remain persistent across multiple sessions or be dismissible by the user.
```

We call the page "Global banner" so i wonder if we should stick with that convention?

Reply:
soft opinion

Reply:
I'd remove the word banner altogether. 

`When naming components within the design system, it is advised to use singular nouns for clarity and consistency. Naming components with singular nouns enhances comprehension and ensures a straightforward understanding of their purpose and grouping. For instance, Global Banner to Global Notification to align seamlessly with other components such as Toast Notification and Inline Notification. This maintains simplicity, aids in intuitive understanding, and fosters a unified language throughout the design system, while reserving the word Banner for further development of new components.`